### ISSUE TITLE
official TWRP for daisy vs. offain's

### ISSUE THREAD

**Issue by letkan** [TeamWin/android_device_xiaomi_daisy#2]
Since the official TWRP-daisy has a maintainer now maybe it's a good idea to merge the 2 versions specially given the fact that all developers build their roms with offain's TWRP in mind. Otherwise the official version has no reason to exist.

**Comment by bigbiff** [comment]
What do you want to merge again?

**Comment by letkan** [comment]
@bigbiff, ATM you cannot use the official TWRP to flash any of the current roms for daisy, the only one that works is offain's (a.k.a. 33bca). Check [here](https://forum.xda-developers.com/mi-a2-lite) for confirmation/experiences.

**Comment by bigbiff** [comment]
Would the builder like to assume maintainership?

**Comment by letkan** [comment]
> Would the builder like to assume maintainership?

That I wouldn't know. [Here](https://github.com/33bca)'s his repo on github, check with him. My point was only that, for the sake of sanity, official TWRP for daisy should be of some use to people wanting to mess with custom roms. Why publish it otherwise?

**Comment by zitronenmelissa** [comment]
Good point, unfortunately official twrp is useless at the moment, even the new version 3.3.1. It gave me error 7 while offain/33bca Version worked without any problems. If he wants to be maintainer, I vote for it.

**Comment by Hopsinkajiri** [comment]
> Vzhledem k tomu, že oficiální TWRP-daisy má nyní správce, možná je dobré sloučit 2 verze speciálně s ohledem na skutečnost, že všichni vývojáři staví své romy s ohledem na offain TWRP. Jinak oficiální verze nemá důvod existovat.
